src/views/HrReports/BloodReports/BloodReports .jsx

```diff
@@ -1,4 +1,4 @@
-import React, { Fragment, useCallback } from 'react'
+import React, { Fragment } from 'react'
 import { useState } from 'react';
 import { useEffect } from 'react';
```

Comment:
multi import from the same package check the weight and use it

Reply:
done